content/en/blog/2022/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-03-15
+slug: k8s-triage-bot-helper-ci-job
+---
+
+# K8s Triage bot Helper CI Job: ‘*make update*’
+
+## Introduction 
+
+If you are contributing to the Kubernetes project and are developing on a Windows PC, it is conceivable that you will encounter certain issues that will cause your patches to master to somehow be halted. This blog is a workaround for a similar issue I encountered when attempting to have my modifications approved and merged into the master repository.
+
+## Why is this needed?
+ 
+While contributing to [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) for some minor documentation changes, the pushed changes needed to be updated with other verified contents of the entire documentation. So, in order for the change to take effect, a single command must be performed to ensure that all tests on the CI pipeline pass. The single command `make update` runs all presubmission verification tests. For some reason on the windows `wsl` environment the tests, specifically the [update-openapi-spec.sh](https://github.com/SubhasmitaSw/kubernetes/blob/master/hack/update-openapi-spec.sh) script, failed (in my case, take a look at the conversation [here](https://github.com/kubernetes/kubernetes/pull/107691)), eventually failing the `pull-kubernetes-verify` tests.
+
+> Advertently, the test cases pass to generate the updated files in other beefy Operating Systems like mac or Linux.
+
+
+### System info:
+
+| OS name | Version | System type |Memory| WSL |
+| ------- | ------- | ----------- | ---  | --- |
+|Windows 11 home|10.0.22000 Build 22000|x64-based|8.00 GB|Ubuntu-20.04 (v2)|
+
+You might encounter the following on your PR
+
+> failing test cases notification from k8s-ci-robot
+
+
+The tests failing the particular issue:
+
+![Failed Test Cases](https://i.imgur.com/pjrfmIY.png)
+
+Consecutively, 
+
+![Failed Test Cases robot comment](https://i.imgur.com/Lxj38kZ.png)
+
+Additionally one can check the failed test via the `link` provided under details in the above image.
+
+### Potential solution
+Run the failing `.sh` scripts individually known from the CI job output, to generate the expected files to fix up the failures. The `.sh` scripts can be found residing under the `hack/` directory at the root of the `kubernetes/kubernetes` code base. 
+
+> kubernetes/kubernetes &rarr; hack/ &rarr; `.sh` update files
```

Comment:
I'm not sure what this is supposed to represent. An excerpt from running `find` would be clearer I think.